.travis.yml

```diff
@@ -34,10 +29,14 @@ jobs:
       install:
       - poetry install
 
+      env:
+      - SECRET_KEY="!pyfta##&3@51#%t=c^7%in1p)n7m^d+(e$!au%*#1m7y5bmr#"
```

Comment:
Put them inside Travis dashboard. It will pick up from there

Reply:
@jsparmani

Reply:
Okay